src/odemis/acq/fastem.py

```diff
@@ -766,6 +766,11 @@ def get_abs_stage_movement(self):
         rel_move_hor = self.field_idx[0] * px_size[0] * field_res[0] * (1 - self._roa.overlap)  # in meter
         rel_move_vert = self.field_idx[1] * px_size[1] * field_res[1] * (1 - self._roa.overlap)  # in meter
 
+        # Acceleration unknown, therefore odemis.util.driver.estimateMoveDuration cannot be used to estimate the time.
```

Comment:
Please look at this function: `util.driver.guessActuatorMoveDuration()`, I guess you could use that one.

Reply:
If I look at `util.driver.guessActuatorMoveDuration()` it takes the speed of the z-axis, independent of which axis you pass. Why? And can I update that code without breaking anything?